Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishDateTimePeriodExtractorConfiguration.cs

```diff
@@ -90,7 +90,7 @@ public bool GetBetweenTokenIndex(string text, out int index)
 
         public bool HasConnectorToken(string text)
         {
-            return text.Equals("and");
+            return text.Equals("and") || text.Equals("through");

```

Comment:
Shouldn't this also use the same regex referenced in EnglishDatePeriodExtractorConfiguration.cs?